test/blackbox/types/HelloWorldTypeObject.cxx

```diff
@@ -115,7 +113,7 @@ const TypeObject* GetMinimalHelloWorldObject()
 
 
     MD5 index_hash("index");
-    for (int i = 0; i < 4; ++i)
+    for(int i = 0; i < 4; ++i)
```

Comment:
This is not from our our c++ style code

Reply:
This file is going to be removed when the TypeObject is generated in my integration branch. I would not be worried.